google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/javadsl/GooglePubSub.scala

```diff
@@ -23,15 +23,27 @@ object GooglePubSub {
 
   /**
    * Creates a flow to that publishes messages to a topic and emits the message ids.
+   * @param overrideHost if present publish message will be sent to specific host,
+   *                     can be used to send message to specific regional endpoint,
+   *                     which can be important when ordering is enabled
    */
   def publish(topic: String,
               config: PubSubConfig,
+              overrideHost: java.util.Optional[String],
```

Comment:
Wouldn't `String` be enough? If you don't want to pass a host you'd use the other overload?

Reply:
Right, changed